shub/config.py

```diff
@@ -131,6 +133,8 @@ def save(self, path=None):
             yml['apikeys'] = self.apikeys
             yml['version'] = self.version
             yml['stacks'] = self.stacks
+            if self.eggs:
+                yml['requirements'] = {'eggs': self.eggs}
```

Comment:
If you can add it for this release, that would be awesome.

On Mon, Sep 5, 2016 at 5:44 PM Jakob de Maeyer <notifications@github.com>
wrote:

> In shub/config.py
> <https://github.com/scrapinghub/shub/pull/201#discussion_r77555003>:
>
> > @@ -131,6 +133,8 @@ def save(self, path=None):
> >              yml['apikeys'] = self.apikeys
> >              yml['version'] = self.version
> >              yml['stacks'] = self.stacks
> > +            if self.eggs:
> > +                yml['requirements'] = {'eggs': self.eggs}
>
> Makes sense to do it in another PR. I'll add one in a bit. Or do you think
> it should also be in a later release?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/scrapinghub/shub/pull/201/files/5065e192fee6a499c39fea80af62f8b0fa37a38e#r77555003>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AACR-cu3ATlSATjmmtpjzy5KWQ64c1LFks5qnH82gaJpZM4Jy5fs>
> .
>